thirdeye/thirdeye-frontend/app/pods/home/controller.js

```diff
@@ -4,6 +4,13 @@ import { computed, get } from '@ember/object';
 
 export default Controller.extend({
 
+  /**
+   * Overrides ember-models-table's css classes
+   */
+  classes: {
+    table: 'table-bordered table-condensed te-anomaly-table--no-margin'
```

Comment:
Is there a reason why we can not do this in the CSS vs here in the Controller? Maybe in the `te-anomaly-table.scss` file? The reason it's not ideal to put css in JS because any updates done in CSS might not catch the ones in JS classes definitions. If we want specific styles for this component or page only, we can assign a specific class name and use that to style in the CSS. Also for searching it's hard to do in both CSS and JS files for any style maintenance. I could be wrong if this is best practices.

Reply:
Based on the documentation for `ember-models-table` (http://onechiporenko.github.io/ember-models-table/v.1/#/), in order to override some default css, I'd have to pass custom classes to the component. It is possible to do in the CSS through, but @justYves has done this in the controller previously here, so I figured it's acceptable: https://github.com/justYves/pinot/blob/4fc67f78c74fe12f9bce95f46609c0930b599aca/thirdeye/thirdeye-frontend/app/pods/components/modals/entity-mapping-modal/component.js#L270. 

Maybe @justYves can also weigh in on this also?

Reply:
Sorry it was late last night i meant not CSS file. But in the HBS. As it's best to keep the separation of concerns with the definitions in CSS and usage in HBS/Template files vs JS. It's not a big thing, I felt it's better if we apply so many classes to a component to do in the template vs the JS. Someone can do a clean up of a class and miss the JS files vs HBS files. If we do this in the JS, it's best to consolidate to a single BEM class perhap.